dask/dataframe/io/tests/test_csv.py

```diff
@@ -431,6 +431,14 @@ def test_read_csv_index():
         assert_eq(result, expected)
 
 
+def test_read_csv_skiprows_range():
+    with filetext(csv_text) as fn:
+        f = dd.read_csv(fn, skiprows=range(5))
+        result = f.compute(scheduler='sync')
```

Comment:
```suggestion
        result = f
```

We prefer to pass in dask objects to `assert_eq` so that it can run some additional checks before computing.